docs/source/getting-started/tabular-data/computing-basic-statistics.ipynb

```diff
@@ -0,0 +1,472 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Computing Basic Statistics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this section, you will learn to compute essential statistical measures such as: \n",
+    "\n",
+    "* Sum \n",
+    "* Mean \n",
+    "* Median\n",
+    "* Quantiles\n",
+    "\n",
+    "For each method, we will compare the actual values to the differentially private values to demonstrate utility. The [documentation](../../api/python/opendp.polars.html) also provides more information about the methods. We will use the [sample data](https://github.com/opendp/dp-test-datasets/blob/master/data/sample_FR_LFS.csv.zip) from the Labour Force Survey in France. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set Up"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import polars as pl \n",
+    "import opendp.prelude as dp\n",
+    "\n",
+    "dp.enable_features(\"contrib\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
I would suggest separating the code from the output explicitly. For example, mentioning that the output is now the following, and then write up the table and all the other stuff, and maybe enclose that in a box, too. For more clarity.

Reply:
just to clarify, so create a new markdown cell and put the string outputs in there directly and recreate the output dataframes in markdown? and take out the results from the cells themselves?

Reply:
Oh, I just meant something really simple. Right now, this output text is just outside the code snippet, and it's kinda hard to see it distinguished from the actual text of the documentation. That's why I suggested using a box or something to enclose it neatly. Additionally, indicating explicitly that this is the output that we would se on executing these scripts would be helpful.

Reply:
gotcha